conans/test/functional/cache/short_paths_test.py

```diff
@@ -162,3 +137,11 @@ def test_leaking_folders(self, use_always_short_paths):
             self.assertEqual(5, len(os.listdir(short_folder)))
             client.run("install dep/1.0@ --build")
             self.assertEqual(5, len(os.listdir(short_folder)))
+
+    def test_info_paths(self):
+        # https://github.com/conan-io/conan/issues/8172
+        client = TestClient()
+        client.save({"conanfile.py": GenConanfile().with_short_paths(True)})
+        client.run("export . test/1.0@")
+        client.run("info test/1.0@ --paths")
+        client.run("info test/1.0@ --paths")
```

Comment:
Do you need to call the same command twice?

Reply:
This is exactly the test. The second one is raising if you do not implement the fix.

Reply:
I can clarify it with a comment

Reply:
ok, no problem, it is said in the issue